blocks/library/button/block.scss

```diff
@@ -17,10 +17,19 @@ $blocks-button__height: 46px;
 	vertical-align: top;
 	position: relative;
 
+	a {
+		color: $white;
+		text-decoration: none;
```

Comment:
Twenty Fifteen uses `border-bottom` for some reason:

![image](https://user-images.githubusercontent.com/134745/27979155-c191ad3a-6328-11e7-8395-b2b2989d41e1.png)

But adding `border-bottom: none` doesn't do the trick since there isn't enough specificity to override the theme's selector:

```
.entry-content a, .entry-summary a, .page-content a, .comment-content a, .pingback .comment-body > a
```